jfix-stdlib-ratelimiter/src/main/kotlin/ru/fix/stdlib/ratelimiter/RateLimiterKt.kt

```diff
@@ -0,0 +1,31 @@
+package ru.fix.stdlib.ratelimiter
+
+import java.time.temporal.TemporalUnit
+
+interface RateLimiterKt : AutoCloseable {
+
+    /**
+     * Get configured rate limit
+     */
+    fun getRate(): Double
+
+    /**
+     * Try to acquire permit without waiting for it to become available.
+     *
+     * @return `true` if permit has been acquired `false` otherwise
+     */
+    suspend fun tryAcquire(): Boolean
```

Comment:
In context of comment for ConfigurableRateLimiterKt regarding throwing exception.

This method `tryAcquire()` without timeout must be non suspendable. It must not block - it just need to return false if permit is not available immediately.

Then we will keep the contract of old java based RateLimiter that we trying to implement here (but in non blocking suspendable fashion)

Reply:
Ok